src/Native/Unix/System.Native/pal_termios.cpp

```diff
@@ -0,0 +1,356 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+#include "pal_types.h"
+#include <termios.h>
+#include <sys/ioctl.h>
+#include <stdio.h>
+
+/* This is dup of System/IO/Ports/NativeMethods.cs */
+enum {
```

Comment:
A nit - the coding convention is to place the `{` on a separate line.

Reply:
Another nit - it seems that it would be nice to name these "ParityXXXX" to match the style of the enum below.

Reply:
nit: either rename as suggested by @janvorli or change NoneParity to NoParity